### PR TITLE
ajikurian add a Dublin hometome file

### PR DESCRIPTION
**Pull request by ajikurian**
This PR will add content for the Dublin hometown file 🌞 

Some open ❓ before I merge:

1. If you haven't been to Dublin, what are you looking forward to?
1. If you have been to Dublin, what did you enjoy? what did you dislike?
1. If you have small children, what sorts of activities do you enjoy doing with them

### CODE REVIEW

**Comment by githubteacher** [comment]
> If you haven't been to Dublin, what are you looking forward to?

@ajikurian I hear the sights are amazing! How's the food though?

File: ajikurian-dublin.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+### Dublin, California
```
Comment (jpadyala) [inline ajikurian-dublin.md]: @ajikurian, I was stayed near Tracy for couple of months, and I like the train commute.

Comment (angpaul2) [inline ajikurian-dublin.md]: @ajikurian, i heard that the daily commute is pretty good :smile:

Comment (ajikurian) [inline ajikurian-dublin.md]: commute is fun 😺

Comment (ajikurian) [inline ajikurian-dublin.md]: @jpadyala train is a good option however not that frequent

**Pull request merged**